sdk/videoanalyzer/azure-media-videoanalyzer-edge/src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.media.videoanalyzer {
+    requires transitive com.azure.core;
+
+    opens com.azure.media.videoanalyzer.edge.models to com.fasterxml.jackson.databind, com.azure.core;
+}
```

Comment:
The file below "samples/README.md" is empty. I couldn't comment on that file, so adding a comment here to add content to it.